simp_le.py

```diff
@@ -81,6 +81,31 @@ class Error(Exception):
     """simp_le error."""
 
 
+_PEM_RE_LABELCHAR = r'[\x21-\x2c\x2e-\x7e]'
+_PEM_RE = re.compile(
+    (r"""
+^-----BEGIN\ ((?:%s(?:[- ]?%s)*)?)\s*-----$
+.*?
+^-----END\ \1-----\s*""" % (_PEM_RE_LABELCHAR, _PEM_RE_LABELCHAR)).encode(),
+    re.DOTALL | re.MULTILINE | re.VERBOSE)
+_PEMS_SEP = b'\n'
+
+
+def split_pems(buf):
```

Comment:
`split_pems()` is now used outside of the `OpenSSLIOPlugin` class so I had to revert 9cf509190faa0c9c78d28559dd2f028a840f088b and make the function useable with string input.